hack/update-deps.sh

```diff
@@ -32,7 +32,7 @@ cd "${ROOT_DIR}"
 # The list of dependencies that we track at HEAD and periodically
 # float forward in this repository.
 FLOATING_DEPS=(
-  "knative.dev/test-infra@${VERSION}"
+  "knative.dev/test-infra@release-0.17"
```

Comment:
That's how they do in eventing too?

Reply:
Yes https://github.com/knative/eventing/pull/3818/files#diff-58d4b0ee04af7710eefa27cf91255798R34